core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/definitions/LdapServicesDecoder.scala

```diff
@@ -222,6 +244,20 @@ object LdapServicesDecoder {
       }
   }
 
+  private implicit val enabledCircuitBreakerDecoder: Decoder[CircuitBreaker.Enabled] =
+    Decoder.forProduct2("max_retries", "reset_duration")(CircuitBreaker.Enabled.apply)
+
+  private lazy val circuitBreakerDecoder: Decoder[CircuitBreaker] =
+    Decoder
+      .instance { c =>
+        val circuitBreaker = c.downField("circuit_breaker")
```

Comment:
* when `circuit_breaker` is not defined, we should enable it with default settings
* when `circuit_breaker` section is malformed, ROR should treat whole config as malformed

Reply:
What should be default settings for CB?

Reply:
let's say:
* maxFailures = 10
* resetDuration = 10 s

Reply:
I gave it more though and I wonder how can we represent disabled CB state in config without requiring other fields present in that state. Will something like that be ok? 
```
circuit_breaker:
  enabled: false (optional)
  max_failures: 1 (required when enabled is true)
  reset_duration: 5s (required when enabled is true)
```

Reply:
Sorry if I jump into this a bit out of the blue, but I was thinking for time measures we could align to the existing naming policy? <what happens after the time>_in_<time unit of measurement> 

As seen in:
* connection_timeout_in_sec
* request_timeout_in_sec
* cache_ttl_in_sec

Reply:
ok, I will make both variants `reset_duration` and `reset_duration_in_sec` allowed

Reply:
@sscarduzio  `_in_sec` convention is deprecated. We maintain it for backward compatibility reasons. 

```
reset_duration: 60 sec
```

or 

```
reset_duration: 2 min
```

It's more elegant and flexible convention. We use it for any duration setting in our config. Maybe we should fix the docs (but there is already jira for this: RORDEV-255)